units/commit_queue.go

```diff
@@ -289,10 +289,9 @@ func (j *commitQueueJob) TryUnstick(ctx context.Context, cq *commitqueue.CommitQ
 				"source":  "commit queue",
 				"job_id":  j.ID(),
 			})
-			j.AddError(model.SetActiveState(mergeTask, evergreen.APIServerTaskActivator, true))
 		}
 		if mergeTask.Blocked() {
-			grip.Critical(message.Fields{
+			grip.Error(message.Fields{
```

Comment:
Same thought here about logging at the error level; these all seem like things we should actually be unsticking as part of the job, or directly notifying the team about